src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -3877,6 +3880,26 @@ static int xmc_access(struct platform_device *pdev, enum xocl_xmc_flags flags)
 		raptor_cmc_access(pdev, flags);
 }
 
+static void clock_status_check(struct platform_device *pdev, bool *latched)
```

Comment:
Same here. Please add some comments about this function. As CMC for U.2 is different, its good to add description about U.2 CMC behavior.